docs/about.html

```diff
@@ -23,10 +23,11 @@
         <section>
             <div class="menu" id="myMenu">
                 <a href="index.html">Home</a>
+                <a href="showcase.html">Our Students</a>
+                <a href="roadmap.html">Education</a>
                 <a href="about.html" class="active">About</a>
-                <a href="roadmap.html">Courses</a>
-                <a href="showcase.html">Students Projects</a>
-                <a href="ev.html">Events</a>
+                <!-- <a href="ev.html">Events</a> -->
+                <a href="https://newtfire.org/courses/DIGITShowcase.html" target="_blank">DIGIT Works</a>
```

Comment:
@SavannahRicks I'm not sure this is the best idea to have the last menu bar item jump straight to my boring newtfire schedule for the current DIGIT Works... Maybe:
* Just keep it as an Events page (b/c we might conceivably want to advertise other events besides DIgit Works)
* Wasn't @rcc5418 going to just put an iframe on the Events page that would provide a view of the latest DIGIT Works schedule, and a means to navigate there?

Reply:
Yeah, we can keep the Events page, I can have the link to your Newtfire <a> tagged in at the bottom of the page

Reply:
I just pulled in the changes and it all looks good! Thanks @SavannahRicks!